polaris-tokens/tokens.ts

```diff
@@ -0,0 +1,75 @@
+import depth from './token-groups/depth.json';
+import legacyTokens from './token-groups/legacy.json';
+import darkColorScheme from './token-groups/color.dark.json';
+import lightColorScheme from './token-groups/color.json';
+import motion from './token-groups/motion.json';
+import shape from './token-groups/shape.json';
+import spacing from './token-groups/spacing.json';
+import typography from './token-groups/typography.json';
+import zIndex from './token-groups/z-index.json';
+import {tokensToRems} from './utilities';
+
+/**
+ * Values to convert to CSS custom properties.
+ * @example {background: '#000'} // --p-background: #000;
+ */
+export interface TokenValues {
+  description: string;
+  figmaName: string;
+  value: string;
+}
```

Comment:
This looks great! Similar to Alex's comment we can likely remove the `figmaName` for now and autogenerate in a build script. (See open-props' [toFigmaTokens transform](https://github.com/argyleink/open-props/blob/a429f931076ab69d3af4539ea312720280cfbdaf/build/props.js#L85-L88))

Also, I do suggest we align our naming with the Design Tokens Glossary as often as possible. In this case I propose updating this type and references to `TokenProperties`: https://www.designtokens.org/glossary/#design-token-properties